app/docs/0.2.x/getting-started/introduction.md

```diff
@@ -8,12 +8,19 @@ title: Welcome to Kong
   <strong>Before you start:</strong> Make sure you've <a href="/install/">installed Kong</a> &mdash; It should only take a minute!
 </div>
 
-Now that you know what Kong is and how it works, this guide is going to take you through a quick introduction to Kong. We'll teach you how to use Kong to:
+Before going further into Kong, make sure you understand its [purpose and philosophy](/about). Once you are confident with the concept of API Gateways, this guide is going to take you through a quick introduction on how to use Kong and perform basic operations such as:
 
-- Running your own Kong instance.
-- Adding and consuming APIs
-- Installing plugins on Kong.
-- *And much more!*
+- [Running your own Kong instance][quickstart].
+- [Adding and consuming APIs][adding-your-api].
+- [Installing plugins on Kong][enabling-plugins].
+
+### What is Kong, technically?
+
+You probably read or heard that Kong was built on top of Nginx. But how is that possible exactly?
```

Comment:
second `Nginx` can be replaced with the letter `a` and avoid duplicate word syndrome.